subsys/settings/Kconfig

```diff
@@ -8,7 +8,9 @@ menuconfig SETTINGS
 	bool
 	default n
 	prompt "Enable settings subsystem with non-volatile storage"
-	depends on FILE_SYSTEM || FCB
+	# Only NFFS is currently supported as FS.
+	# The reason ins that FatFs doesn't implement the fs_rename() API
```

Comment:
ins -> is

Reply:
That was copy-paste from your suggestion ;) Fixing in a moment.

Reply:
Fixed now